Remotion/Web/Development.WebTesting/WebDriver/Configuration/Firefox/FirefoxConfiguration.cs

```diff
@@ -111,7 +111,7 @@ public virtual FirefoxOptions CreateFirefoxOptions ()
       var firefoxOptions =  new FirefoxOptions
              {
                  Profile = profile,
-                 BrowserExecutableLocation = BrowserBinaryPath,
+                 BinaryLocation = BrowserBinaryPath,
```

Comment:
@Frazerus  you think this might help with Edge v121?

Reply:
Great question, will check out :)